control/control.CAASP.xml

```diff
@@ -275,6 +275,97 @@ textdomain="control"
+        </software>
+        <partitioning>
+            <expert_partitioner_warning config:type="boolean">true</expert_partitioner_warning>
+
+            <proposal>
+                <lvm config:type="boolean">false</lvm>
+                <proposal_settings_editable config:type="boolean">false</proposal_settings_editable>
+                <root_subvolume_read_only config:type="boolean">true</root_subvolume_read_only>
+            </proposal>
+
+            <volumes config:type="list">
+                <!-- The / filesystem -->
+                <volume>
+                    <mount_point>/</mount_point>
+                    <!-- Default == final, since the user can't change it -->
+                    <fs_type>btrfs</fs_type>
+                    <desired_size config:type="disksize">25 GiB</desired_size>
+                    <min_size config:type="disksize">16 GiB</min_size>
+                    <max_size config:type="disksize">30 GiB</max_size>
+                    <weight config:type="integer">80</weight>
+                    <!-- Always use snapshots, no matter what -->
+                    <snapshots config:type="boolean">true</snapshots>
+                    <snapshots_configurable config:type="boolean">false</snapshots_configurable>
+
+                    <!-- You don't want to miss the / volume -->
+                    <proposed_configurable config:type="boolean">false</proposed_configurable>
+
+                    <!-- the default subvolume "@" was requested by product management -->
+                    <btrfs_default_subvolume>@</btrfs_default_subvolume>
+
+                    <!-- Subvolumes to be created for a Btrfs root file system -->
+                    <!-- copy_on_write is true by default -->
+                    <!-- The XML parser is very simplistic, thus not using attributes, but sub-elements -->
+                    <subvolumes config:type="list">
+                        <subvolume>
+                            <path>root</path>
+                        </subvolume>
+                        <subvolume>
+                            <path>tmp</path>
+                        </subvolume>
+                        <subvolume>
+                            <path>home</path>
+                        </subvolume>
+                        <subvolume>
+                            <path>cloud-init-config</path>
+                        </subvolume>
+                        <!-- unified var subvolume - https://lists.opensuse.org/opensuse-packaging/2017-11/msg00017.html -->
+                        <subvolume>
+                            <path>var</path>
+                            <copy_on_write config:type="boolean">false</copy_on_write>
+                        </subvolume>
+
+                        <!-- architecture specific subvolumes -->
+
+                        <subvolume>
+                            <path>boot/grub2/arm64-efi</path>
+                            <archs>aarch64</archs>
```

Comment:
do we plan caasp for other arch? or it is just for safe?

Reply:
It's the same as the default proposal. Better be safe here...